docs/OrderUsage.md

```diff
@@ -0,0 +1,176 @@
+  "time": "2022-03-16T11:13:32.571Z",
+  "v": 0
+}
+```
+
+## How to fetch subscription-orders ?
+
+You might need to fetch subscription-order history, to support viewing and editing the subscription order.
+
+The Order search query example:
+
+```json
+{
+  "query": {
+    "exists": {
+      "field": "custom.deliveryDate",
+      "customType": "StringType"
+    }
+  }
+}
+```
+
+> Note that order search API does not support filtering custom fields based on the type `ReferenceField` yet, so in this query we have used the `deliveryDate` instead.
```

Comment:
It is not clear why you are talking about `ReferenceField` here. Probably you wanted to talk about why `subscriptionTemplateOrderRef` attr is not used?

Reply:
Check this one: https://github.com/commercetools/commercetools-subscriptions/pull/17/commits/7e8def69a9f6c3a189c0bb8547b15350d9c233c8